Dockerfile

```
@@ -41,6 +41,7 @@ RUN echo "**** install dev packages ****" && \
     mamba install --quiet \
         git \
         python="3.10" \
+        "setuptools<66.0.0" \
```

```suggestion
        "setuptools<66.0.0a" \
```

This should be temporary but works for now.